interfaces/json_test.go

```diff
@@ -29,94 +29,44 @@ import (
 )
 
 type JSONSuite struct {
-	plug *Plug
-	slot *Slot
+	plug *snap.PlugInfo
+	slot *snap.SlotInfo
 }
 
 var _ = Suite(&JSONSuite{
-	plug: &Plug{
-		PlugInfo: &snap.PlugInfo{
-			Snap:      &snap.Info{SuggestedName: "snap-name"},
-			Name:      "plug-name",
-			Interface: "interface",
-			Attrs:     map[string]interface{}{"key": "value"},
-			Apps: map[string]*snap.AppInfo{
-				"app-name": {
-					Name: "app-name",
-				},
+	plug: &snap.PlugInfo{
+		Snap:      &snap.Info{SuggestedName: "snap-name"},
+		Name:      "plug-name",
+		Interface: "interface",
+		Attrs:     map[string]interface{}{"key": "value"},
+		Apps: map[string]*snap.AppInfo{
+			"app-name": {
+				Name: "app-name",
 			},
-			Label: "label",
 		},
-		Connections: []SlotRef{{
-			Snap: "other-snap-name",
-			Name: "slot-name",
-		}},
+		Label: "label",
```

Comment:
The label was associated with AppInfo before, now it looks like with PlugInfo, is this intentional?